src/webnn_native/ops/LeakyReluBase.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 The WebNN-native Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef WEBNN_NATIVE_OPS_LEAKYRELUBASE_H_
+#define WEBNN_NATIVE_OPS_LEAKYRELUBASE_H_
+
+#include "webnn_native/ops/ClampBase.h"
+
+namespace webnn_native { namespace op {
+
+    class LeakyReluBase {
```

Comment:
The base file is simple to add the options in `Operator`, it's unnecessary add a new file.

Reply:
LeakyReluOperator and LeakyRelu both need to use GetAlpha() function, so I add a base file. What's your opinion? Thanks! @fujunwei

Reply:
I suggest moving to` LeakyReluOperator and LeakyRelu`

Reply:
Hi, ningxin, I added ClampBase and LeakyReluBase for sharing same codes such as GetOptions() and GetAlpha(), does it align with your suggestions as below?  @huningxin 
> Could `LeakyReluOperator` share implementation `LeakyRelu`? They share same code such as alpha setting etc.,.

Reply:
I suppose it would be good to share the `LeakyReluOptions` handling code between operand and operator. I agree we should avoid add too many small files. How about combine the LeaklyReluBase, LeaklyReluOperator and LeaklyReluOperand into one LeaklyRelu.h file. Any thoughts @fujunwei ?